analysis/cross_partition_combiners.py

```diff
@@ -83,6 +83,12 @@ def _sum_metrics_to_value_error(sum_metrics: metrics.SumMetrics,
         l1=l1,
         rmse_with_dropped_partitions=rmse_with_dropped_partitions,
         l1_with_dropped_partitions=l1_with_dropped_partitions)
+    if keep_prob != 1:
```

Comment:
Is the multiplication really important for performance here? If not I think we can simply skip (especially given that keep_prob=1 only when public partitions are used (?)

Reply:
yes, multiplication for performance reasons. And I'd prefer to keep it. `keep_prob = 1` for any large enough partition. I'